proxy-manager/DOCS.md

```diff
@@ -82,8 +83,16 @@ cleared automatically_
   by taking a snapshot of just the Nginx Proxy Manager add-on, will not
   contain the data from this add-on. Please make sure you backup both
   the Nginx Proxy Manager and the MariaDB add-ons.
-
-## Changelog & Releases
+- As of Home Assitant Core 2021.07 unconfigured reverse proxies are blocked with a `400` http response.  
+  You may specify the default docker network to trust NPM, and all containers running on your host.
+  ```
+   # enable https  proxy
+   http:
+     use_x_forwarded_for: true
+     trusted_proxies: [ 172.30.33.0/24 ]
```

Comment:
- Please use block-style sequences and not flow style.
- Don't expose the full Docker network; from a security perspective, that is a bad practice.

Reply:
Am I right that we should be able use the NPM container name here instead?